cli/src/native.rs

```diff
@@ -84,6 +88,37 @@ pub(crate) async fn run(args: Params) -> Result<()> {
     }
 }
 
+async fn open_db(path: Option<PathBuf>, network_id: &str) -> Result<Db> {
+    if let Some(path) = path {
+        let db = spawn_blocking(|| sled::open(path)).await??;
+        return Ok(db);
+    }
+
+    let cache_dir =
+        ProjectDirs::from("co", "eiger", "lumina").context("Couldn't find lumina's cache dir")?;
+    let mut cache_dir = cache_dir.cache_dir().to_owned();
+
+    // TODO: remove it after we publish a new version to crates.io?
```

Comment:
I think we might need to leave it up for few versions, for everybody to get a chance to run the version with migration code. Not sure how important is it? We lose few hours of download, which is bad, but not terrible.

Reply:
changed the todo